Sources/Aardvark/BugReporting/ARKBugReporter.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Copyright 2021 Square, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import CoreAardvark
+
+@objc public protocol ARKBugReporter: NSObjectProtocol {
+
+    /// Called when the user has triggered the creation of a bug report, including a screenshot attached to the report.
+    func composeBugReport()
+
+    /// Called when the user has triggered the creation of a bug report, without an attachment screenshot.
+    func composeBugReportWithoutScreenshot()
+
+    /// Add logs from `logStores` to future bug reports.
+    @objc(addLogStores:)
+    func add(logStores: [ARKLogStore])
+
+    /// Remove logs from `logStores` from future bug reports.
+    @objc(removeLogStores:)
+    func remove(logStores: [ARKLogStore])
+
+    /// The log stores used to generate bug reports.
+    func logStores() -> [ARKLogStore]
```

Comment:
Could this be a computed property?
```suggestion
    var logStores: [ARKLogStore] { get }
```

I don't think that'd make the change a breaking one since the objc call-site would still be of the form `[bugReporter logStores];`

Reply:
It would be a breaking change for Swift call sites.

Reply:
well that's a good point.